<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAQDeviceFunctionExtractionWizardPage3GetData</class>
 <widget class="QWizardPage" name="DAQDeviceFunctionExtractionWizardPage3GetData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="QCustomPlot" name="plotBuilder" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="getDataHorizontalLayout">
     <item>
      <widget class="QPushButton" name="startPushButton">
       <property name="text">
        <string>Начать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="endPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Остановить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
